packages/recomponents/src/components/r-button/r-button.scss

```diff
@@ -18,6 +17,13 @@
     outline: none;
     transition: var(--button-transition);
 
+    .capitalize-the-first-letter {
```

Comment:
I think this should be renamed to `capitalize-first-letter`, the `the` is overly verbose

also, please add this CSS to the `helpers` file instead